vespajlib/src/main/java/ai/vespa/http/DomainName.java

```diff
@@ -15,8 +15,8 @@
  */
 public class DomainName extends PatternedStringWrapper<DomainName> {
 
-    static final Pattern labelPattern = Pattern.compile("([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]{0,61}[A-Za-z0-9])");
-    static final Pattern domainNamePattern = Pattern.compile("(" + labelPattern + "\\.)*" + labelPattern);
+    public static final Pattern labelPattern = Pattern.compile("([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]{0,61}[A-Za-z0-9])");
```

Comment:
Doesn't look like these need to be public?

Reply:
Not the label. The domain pattern is used by `HostName`. It's probably better to just replicate the pattern. Will do that.